index.js

```diff
@@ -1,63 +1,80 @@
-const books = [
-  {
-    title: 'Life and Living',
-    author: 'John Doe',
-    id: 1,
-  },
-  {
-    title: 'Becoming',
-    author: 'Michelle Obama',
-    id: 2,
-  },
-];
-
-const bookListSection = document.querySelector('#book-list');
-
-function displayBooksList(bookList) {
-  bookListSection.innerHTML = bookList.map((book) => `
-<p class="title">${book.title}</p>
-            <p>${book.author}</p>
-            <button data-id=${book.id} class="remove">Remove</button>
-            <hr>`).join('');
-}
+const bookList = document.querySelector('.books');
+const form = document.getElementById('add-book');
+const titleInput = document.getElementById('title');
+const authorInput = document.getElementById('author');
 
-function storeBook(bookList) {
-  localStorage.setItem('bookList', JSON.stringify(bookList));
-}
+let books = [];
+
+class BookClass {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+
+  bookCode() {
+    return `<div class="book-block">
+    
+                <div class="title-author">
+
+                    <div class="title">${this.title}</div>
+                    <div> <p>by</p> </div>
+                    <div>${this.author}</div>
+
+                </div>
+
+                <div class="buttons">
+                  <button data-id=${this.id} class="remove">Remove</button>
+                </div>
+            </div>`;
+  }
+
+  static addBook(book) {
+    let id = 1;
+    if (books.length > 0) {
+      id = books[books.length - 1].id + 1;
+    }
+    book.id = id;
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
```

Comment:
- [ ]  You should have class methods to add and remove books as well as class attributes like the books array, which means that those must be inside the `books class`. Let's fix that